### PR TITLE
fix Issues regarding to Methods Naming Conventions

### PR DESCRIPTION
**Pull request by tsvetaevAlex**

### CODE REVIEW

File: jdi-bdd-tests/src/main/java/io/github/com/custom/MenuItem.java [hunk 1]
```diff
@@ -1,8 +1,8 @@
 package io.github.com.custom;
 
-import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.common.uiElement;
 
-public class MenuItem extends UIElement {
+public class MenuItem extends uiElement {
```
Comment (evasileva-epam) [inline jdi-bdd-tests/src/main/java/io/github/com/custom/MenuItem.java:5]: No need in this changes. Please see https://www.cs.rice.edu/~cork/book/node15.html#:~:text=By%20convention%2C%20Java%20programs%20are,are%20written%20entirely%20capital%20letters.

File: jdi-bdd/src/main/java/com/epam/jdi/bdd/elements/CustomMenu.java [hunk 2]
```diff
@@ -1,11 +1,11 @@
 package com.epam.jdi.bdd.elements;
 
 import com.epam.jdi.light.elements.complex.Menu;
-import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.complex.webList;
 
 public class CustomMenu extends Menu {
     @Override
-    public WebList list() {
+    public webList list() {
```
Comment (evasileva-epam) [inline jdi-bdd/src/main/java/com/epam/jdi/bdd/elements/CustomMenu.java:8]: The same for WebList as for UIElement

File: jdi-bdd-tests/src/main/java/io/github/com/custom/MenuItem.java (hunk 1 above)
Comment (tsvetaevAlex) [reply jdi-bdd-tests/src/main/java/io/github/com/custom/MenuItem.java]: fixed

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Carousel.java [hunk 3]
```diff
@@ -35,7 +35,8 @@
 	protected String indicatorsLocator = "li[data-slide-to]";
 
 	@UI(".carousel-item") @GetAny public DataList<T, D> slides;
-	@UI(".carousel-item") @VisualCheck UIElement activeSlide;
+	@UI(".carousel-item") @VisualCheck
+    UIElement activeSlide;
```
Comment (evasileva-epam) [inline jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Carousel.java:39]: Do you really need this changes? The same for jdi-light-html-tests/src/main/java/pseudo/site/forms/CustomForm.java

**Review by evasileva-epam — CHANGES_REQUESTED** [review]

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Carousel.java (hunk 3 above)
Comment (tsvetaevAlex) [reply jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Carousel.java]: extra newLine removed

File: jdi-light-html-tests/src/main/java/pseudo/site/forms/CustomForm.java [hunk 4]
```diff
@@ -32,12 +32,15 @@
     @Css(".cssWebElementList-class") List<WebElement> cssWebElementList;
     @XPath("//*[@class='xpathPublicWebElementList-class']") public List<WebElement> xpathPublicWebElementList;
 
-    @FindBy(css = "#fbUIElement-id") UIElement fbUIElement;
+    @FindBy(css = "#fbUIElement-id")
+    UIElement fbUIElement;
```
Comment (evasileva-epam) [inline jdi-light-html-tests/src/main/java/pseudo/site/forms/CustomForm.java:36]: Haven't updated CustomForm.java as it was asked in previous comment

**Review by evasileva-epam — CHANGES_REQUESTED** [review]

**Pull request closed without merging**